src/test/java/com/google/sps/servlets/DevicesServletTest.java

```diff
@@ -43,6 +44,11 @@
   private final String TEST_USER_EMAIL = "testEmail";
   private final String TEST_USER_AUTH_DOMAIN = "testAuthDomain";
 
+  private final String TEST_MAX_COUNT = "maxCount";
```

Comment:
I suppose the value does not matter to the tests because it is used as a stub, but perhaps we can set it to something more meaningful (e.g. "300") so people looking at the tests can see what proper usage looks like.  This way the tests help document proper usage.  Possibly consider doing this for some of the other test values too.

Reply:
300 is a magic number that I would not want to introduce. I believe that the names themselves are clear enough to be self-documenting.